src/main/java/Lotes.java

```diff
@@ -0,0 +1,108 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+public class Lotes {
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+    private static final String lineSeparator = System.lineSeparator();
+    private static final String horizontalLine = "    _________________________"
+                                                + "___________________________";
+
+    private static void printTasksList() {
+        System.out.println(horizontalLine);
+
+        if (taskList.isEmpty()) {
+            System.out.println("     List is empty, please enter some text to add to list.");
+        } else {
+            System.out.println("     Here are the tasks in your list:");
+
+            int index = 0;
+            for (Task task : taskList) {
+                index++;
+                System.out.print("     " + index + ".[" + task.getStatusIcon()
```

Comment:
Perhaps try Overriding the toString() method in task to reduce the number of lines here